packages/styling/src/interfaces/IFontStyles.ts

```diff
@@ -4,16 +4,16 @@ import { IRawStyle } from '../interfaces/index';
  * UI Fabric font set.
  */
 export interface IFontStyles {
```

Comment:
so loadTheme takes ITheme. It support partials. It should be a partial theme with partial/optional fonts and colors. We mix them on top of the current theme.

```ts
loadTheme({ palette: { themePrimary: 'red' } });
```